modules/pushsnap_scriptworker/files/requirements.txt

```diff
@@ -3,31 +3,30 @@
 # Please do not update this file without end-to-end testing. pushsnapscript has a major hack in its
 # dependencies. Please read https://bugzilla.mozilla.org/show_bug.cgi?id=1447263#c15 for context.
 PyYAML==3.13
-aiohttp==3.4.4
-arrow==0.12.1
+aiohttp==3.5.4
+arrow==0.13.1
 asn1crypto==0.24.0
-async_timeout==3.0.0
-attrs==18.2.0
+async_timeout==3.0.1
+attrs==19.1.0
 certifi==2018.11.29
-cffi==1.11.5
+cffi==1.12.2
 chardet==3.0.4
-click==6.7
-# Bug 1507497 - Cryptography unable to update past 2.3.x on scriptworkers
-cryptography==2.3.1  # pyup: ignore
+click==7.0
```

Comment:
I'm concerned about this bump. Indeed, the package we use specify `>= 6.2` https://github.com/snapcore/snapcraft/blob/2.43.1/requirements.txt#L1, but latest revisions show `~= 6.0` https://github.com/snapcore/snapcraft/blob/ade7d65f0ba7870add2ed5ba5c016a93db2f88b8/requirements.txt#L1.

Is there another package needing to use 7.0?

Reply:
As mentioned in channel, pinning this to 6.7